src/protocols/utils/dist.c

```diff
@@ -71,14 +71,13 @@ int nn_dist_send (struct nn_dist *self, struct nn_msg *msg,
     struct nn_dist_data *data;
     struct nn_msg copy;
 
-    /*  TODO: We can optimise for the case when there's only one outbound
+    /*  TODO: We can optimize for the case when there's only one outbound
```

Comment:
euro vs. us english.  even though i'm in the us, please don't bother to change these.